op-simulator/op_simulator.go

```diff
@@ -0,0 +1,99 @@
+package op_simulator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strconv"
+	"sync/atomic"
+
+	ophttp "github.com/ethereum-optimism/optimism/op-service/httputil"
+	"github.com/ethereum-optimism/supersim/anvil"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type Config struct {
+	Port uint64
+}
+
+type OpSimulator struct {
+	log        log.Logger
+	anvil      *anvil.Anvil
+	HttpServer *ophttp.HTTPServer
```

Comment:
any reason `HttpServer` is exposed as a public field? Seems like it's not used elsewhere

Reply:
whoops, should not be public